dropshot_endpoint/Cargo.toml

```diff
@@ -13,12 +13,15 @@ proc-macro = true
 [dependencies]
 proc-macro2 = "1"
 quote = "1"
-serde_tokenstream = "0.1.3"
+serde_tokenstream = "0.1"
 
 [dependencies.serde]
 version = "1.0"
 features = [ "derive" ]
 
 [dependencies.syn]
-version = "1.0.85"
+version = "1.0"
```

Comment:
This is basically unrelated: I'm all for this but FYI I think it's dependabot that's replacing these with more specific versions when it updates deps.  It doesn't always do this and I'm not sure exactly when it does.  If the Renovate experiment goes well I hope it will give us more control over this.